src/pages/timer.js

```diff
@@ -0,0 +1,107 @@
+import {useState, useEffect} from 'react';
+
+import CookingTimer from '../components/CookingTimer';
+import {StyledInput, StyledInputTime} from '../components/styled/StyledInput';
+
+function Timer() {
+	const [timerDays, setTimerDays] = useState();
+	const [timerHours, setTimerHours] = useState();
+	const [timerMinutes, setTimerMinutes] = useState();
+	const [timerSeconds, setTimerSeconds] = useState();
+	const [userInputDays, setUserInputDays] = useState(0);
+	const [userInputHours, setUserInputHours] = useState(0);
+	const [userInputMinutes, setUserInputMinutes] = useState(0);
+	const [userInputSeconds, setUserInputSeconds] = useState(0);
+
+	//Ich möchte dass der User z.B. eine Stundenanzahl eintragen kann, und dann der Countdown von dem Wert (z.B. 8h30min) runterzählt evtl statt 'October,8...' einfach Date.now() + (UserInput) = deadline ? und danach bei getTime deadline - date.now
+
+	//Fazit: ------>  -V- Date.now() + (UserTimeInput) = deadline
+	const seconds = userInputSeconds * 1000;
+	const minutes = userInputMinutes * 60000;
+	const hours = userInputHours * (60000 * 60);
+	const days = userInputDays * (60000 * 60 * 24);
+	//const deadline = '10, 9, 2022 00:15:00 GMT';
```

Comment:
also delete code you dont need anymore. 
please review all your code for unneccessary comments.